### PR TITLE
add more test coverage for react

### PR DESCRIPTION
**Pull request by ruddell**
This gets all of the reducer specs to near 100%

@deepu105 The only component tests for React are for the Navbar, the other tests are service and reducer tests.  Is this something we want?

- Please make sure the below checklist is followed for Pull Requests.

- [x] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
- [x] Tests are added where necessary
- [x] Documentation is added/updated where necessary
- [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: generators/client/templates/react/src/test/javascript/spec/app/shared/reducers/user-management.spec.ts.ejs [hunk 1]
```diff
@@ -1,99 +0,0 @@
-<%#
```
Comment (ruddell) [inline generators/client/templates/react/src/test/javascript/spec/app/shared/reducers/user-management.spec.ts.ejs]: This file wasn't copied into the client, there is another user-management.spec.ts file [here](https://github.com/jhipster/generator-jhipster/blob/f1c05547f556c371adb3832fc2e7ed0bdf5b4919/generators/client/templates/react/src/test/javascript/spec/app/modules/administration/user-management/user-management.reducer.spec.ts.ejs)

Comment (vishal423) [reply generators/client/templates/react/src/test/javascript/spec/app/shared/reducers/user-management.spec.ts.ejs]: I may be wrong, but, think this was for OIDC flow.

Comment (ruddell) [reply generators/client/templates/react/src/test/javascript/spec/app/shared/reducers/user-management.spec.ts.ejs]: Can't believe I missed that

**Comment by deepu105** [comment]
@ruddell ofcourse it would be nice to have more component tests

**Pull request merged**